local/languages/finna/en-gb.ini

```diff
@@ -1348,6 +1349,7 @@ hierarchy_view_context = "Show archive structure"
 hierarchy_show_tree = "Show full structure"
 hierarchy_hide_tree = "Hide full structure"
 in_collection_label = "In collection:"
+Included in the collections = "Included in the collections"
```

Comment:
Miksi tässä on "the"? Vaikuttaa tarpeettomalta, eikä vastaavasti "Contains collections"-käännöksessä ole.